continuous-testing/README.md

```diff
@@ -4,24 +4,24 @@ This this guide, we explore best practices for continuous testing - including ex
 
 ## Use Cases
 
-In this section, we provide information about potentially useful *use cases* for continuous testing. The key idea is to provide some more nuanced context for the use cases such that technology (tool) solutions can be shared in context. 
+In this section, we provide information about potentially useful *use cases* for continuous integration. The key idea is to provide some more nuanced context for the use cases such that technology (tool) solutions can be shared in context. 
```

Comment:
We don't want to use the term "integration" here or throughout this README, please use "testing".

Reply:
Fixed.